back/app/serializers/web_api/v1/custom_field_serializer.rb

```diff
@@ -20,6 +20,10 @@ class WebApi::V1::CustomFieldSerializer < WebApi::V1::BaseSerializer
     object.input_type == 'page'
   }
 
+  attribute :dropdown_layout, if: proc { |object, _params|
+    %w[multiselect select].include?(object.input_type)
+  }
```

Comment:
Minor point, but I see this in 2 places in this PR.

Ideally, we should have one source of truth, to avoid these getting out of sync.

Maybe a new model method;
```Ruby
def dropdown_layout_type?
  %w[multiselect select].include?(input_type)
end
```
+ a test in [custom_field_spec.rb](https://github.com/CitizenLabDotCo/citizenlab/blob/master/back/spec/models/custom_field_spec.rb)

Then, here we could use `object.dropdown_layout_type?`